packages/react/src/utils/withRef.ts

```diff
@@ -6,6 +6,10 @@ import { FC, forwardRef } from 'react';
  * @param component Component to `forwardRef`.
  */
 /* eslint-disable @typescript-eslint/no-explicit-any */
-export const withRef = <C extends FC<any>>(component: C): C =>
+export const withRef = <C extends FC<any>>(component: C): C & Named =>
   forwardRef(component as any) as any;
 /* eslint-enable @typescript-eslint/no-explicit-any */
+
+interface Named {
+  displayName?: string;
```

Comment:
Could this be not optional?

Reply:
It could, but it wouldn't be accurate, because the return from `withRef` has an undefined `displayName`.